app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -225,6 +226,15 @@ class ChatRoomPresenter @Inject constructor(
                 client.messages(chatRoomId, roomTypeOf(chatRoomType), offset, 30).result
             }
         messagesRepository.saveAll(messages)
+
+        //if success - saving last synced time
+        if (messages.isEmpty()) {
+            messagesRepository.saveLastSyncDate(System.currentTimeMillis())
```

Comment:
we call saveLastSyncDate() inside loadAndShowMessages() method because it's the plain request for the latest chat history